app/views/catalog/_mapped_results_sidebar.html.erb

```diff
@@ -0,0 +1,16 @@
+<% if @document_list.length > 0 %>
+  <h6 class="nomar-t">Mapped Results:
+          <%= (@response.params[:start].to_i + 1).to_s + ' - ' +
+          (@response.params[:start].to_i + @document_list.length).to_s  %>
+  </h6>
+<% end %>
+<script>
+  DCV.mapCoordinateSearchUrl = decodeURIComponent('<%= CGI.escape(url_for(params.except(:utf8).merge(:lat => '_lat_', :long => '_long_', :search_field => params[:search_field] || 'all_text_teim', :q => params[:q] || ''))).html_safe %>');
+</script>
+<% map_search_settings = controller.subsite_config.fetch('map_search', {}) %>
```

Comment:
let's open a ticket to clean this up later - it'd be good not to have quite so many defaults and OR's here